shared/team-building/user-bubble.js

```diff
@@ -14,7 +15,13 @@ export type Props = {
 
 const KeybaseUserBubble = (props: Props) => (
   <Kb.Box2 className="user" direction="horizontal" style={styles.bubble}>
-    <Kb.Avatar size={bubbleSize} username={props.username} />
+    <Kb.ConnectedNameWithIcon
+      colorFollowing={true}
+      horizontal={false}
+      icon={props.service !== 'keybase' ? serviceIdToIconFont(props.service) : undefined}
```

Comment:
service icons don't quite fit
<img width="198" alt="image" src="https://user-images.githubusercontent.com/11968340/53757403-8c7db100-3e89-11e9-8506-fa0eeada907a.png">

Reply:
@chrisnojima I wasn't able to figure out why the service icons get chopped off. I asked @buoyad and he was stumped too, can you help figure that part out?